Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/httpbody/CultureItem.java

```diff
@@ -66,6 +66,14 @@
     @Expose
     private boolean publicAccessibility;
 
+    @SerializedName("is_favorite")
```

Comment:
When you add a new field to CultureItem, you have to change the Parcelable implementation to parcel this field as well. To do this for ```isFavorite```,
1. you need to read a boolean into ```isFavorite``` in ```CultureItem(Parcel in)``` constructor,
2. you need to write a boolean value into ```out``` Parcel in ```writeToParcel``` function.
3. The order in which you read from parcel, and you write to parcel must be the same, i.e. if you read into ```isFavorite``` after reading into ```publicAccessibility```, you need to write into ```isFavorite```  after you write into ```publicAccessibility```.

Reply:
done @eozd